packages/open-api-gateway/src/project/codegen/components/client-settings.ts

```diff
@@ -92,92 +92,50 @@ export class ClientSettings extends Component {
       options.generatedCodeDir,
       this.clientSettingsFilename
     );
-    // add client settings file to .gitignore
-    this.project.addGitIgnore(this.clientSettingsPath);
```

Comment:
Dont we still need this?

Reply:
No - I got rid of this because the generated clients are checked in, so if we also check this in we save a redundant regeneration of clients on first checkout and build :)

It's also useful to check this in so that we can implement cleanup in future - ie if you remove a generated client from the project definition we can see that it was previously there and delete the appropriate generated client directory :)

Reply:
OK, sgtm